rpm/sailfish-secrets.spec

```diff
@@ -400,6 +400,7 @@ ln -s ../sailfish-secretsd.service %{buildroot}/%{user_unitdir}/user-session.tar
 
 %files -n libsailfishcrypto-tests
 %defattr(-,root,root,-)
+%{_bindir}/sailfishcryptoexample
```

Comment:
it actually belongs in a separate examples package, but I'm not sure how much value there is in adding yet another package to this spec, for this example which is primarily for documentation purposes anyway...